src/core/plugin/luapi_application.h

```diff
@@ -813,9 +813,46 @@ static int applib_addStrokes(lua_State* L) {
 }
 
 /**
- * Puts a Lua Table of the coordinates of the points of all strokes from the selected layer/selection tool onto the
- * stack
- * TODO document the returned table and the parameters
+ * Puts a Lua Table of the Strokes (from the selection tool / selected layer) onto the stack.
+ * Is inverse to app.addStrokes
+ *
+ * Required argument: type ("Selection" or "Layer")
+ *
+ * Example:
+ *
+ * app.getStrokes() = {
+ *         {   -- Inside a stroke are three tables of equivalent length that represent a series of points
```

Comment:
This should be expressed slightly differently, since the examples thus far stand for code that you can execute. Here the right side is just a possible result of that code.

Reply:
You're right, this notation is a bit too short. Should be fixed now